std_msgs/Header header
TacVector[] sensorsArray
